plugins/ScheduledReports/API.php

```diff
@@ -608,10 +609,9 @@ public function sendReport($idReport, $period = false, $date = false, $force = f
         $now = Date::now()->getDatetime();
         $this->getModel()->updateReport($report['idreport'], array('ts_last_sent' => $now));
 
-        // If running from piwik.php with debug, do not delete the PDF after sending the email
-        $tracker = new Tracker();
-        if (!$tracker->isDebugModeEnabled()) {
+        if (!Development::isEnabled()) {
             @chmod($outputFilename, 0600);
+            @unlink($outputFilename);
```

Comment:
Can we use Filesystem::deleteFileIfExists ? Always good to use our own API's so changes can be made in one place if needed. Is the chmod needed at all? Is it in case the file was not deleted?